### PR TITLE
[156]Style the Contact Us Page using CSS -Frontend

### PR DESCRIPTION
**Pull request by Lillian-1**
Screenshot 2020-12-16 143105
Screenshot 2020-12-16 143136
Screenshot 2020-12-16 143236
Screenshot 2020-12-16 143301
Screenshot 2020-12-16 143326
Screenshot 2020-12-16 143352
Screenshot 2020-12-16 143409

### CODE REVIEW

File: submissions/lillianNwaoha/html/contact.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        </label> <br> <br>
+        <label for="subject">
+            <input type="text" placeholder="subject">
+        </label> <br> <br>
+        
+        <label for="message">message <br>
+            <input type="number" class="number" min="0" max="200">
+        </label> <br> <br>
+       <button>Send</button>
+       </form>
```
Comment (Ireti203) [inline submissions/lillianNwaoha/html/contact.html:60]: Remove the breaks in your code, style With CSS instead

**Review by Ireti203 — CHANGES_REQUESTED** [review]
You have 2 contact.html files. One in the html folder and the other in the html-forms folder. Kindly remove the one that is not being used.

**Comment by Ireti203** [comment]
The logo on the navbar for laptop view looks quite distant from the right end of the navbar. Kindly adjust this in your styling.

**Comment by Lillian-1** [comment]
Done


Sent from Yahoo Mail for iPhone


On Wednesday, December 16, 2020, 3:11 PM, Ireti203 <notifications@github.com> wrote:




The logo on the navbar for laptop view looks quite distant from the right end of the navbar. Kindly adjust this in your styling.

**Review by adaugochi — APPROVED** [review]

**Pull request merged**